terraform/modules/hub/files/matomo/matomo-adhoc-def.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "cpu": 512,
+    "memory": 512,
+    "essential": true,
+    "image": "${image_and_tag}",
+    "name": "matomo-adhoc",
```

Comment:
don't think this file is needed any more (non-blocker)

Reply:
gone